.github/ISSUE_TEMPLATE/6_infrastructure.yml

```diff
@@ -1,3 +1,4 @@
+# TBD - keeping the issue template?
```

Comment:
Keep or go on this template?

Reply:
Ultimately get rid of (or tweak) it, but I don't want to disrupt Kent's workflow by removing so I'll take a look this week.

Reply:
Filed https://github.com/flutter/flutter/issues/165814.

Reply:
Fab thanks! I will update with a todo referencing the issue